STPopup/STPopupController.h

```diff
@@ -174,14 +180,6 @@ typedef NS_ENUM(NSUInteger, STPopupTransitionStyle) {
  */
 - (void)popViewControllerAnimated:(BOOL)animated;
 
-
-/**
- Pops all view controllers from the stack until it reaches the root view controller
-
- @param animated YES if animated
- */
-- (void)popToRootViewControllerAnimated:(BOOL)animated;
```

Comment:
Why is this method removed?

Reply:
oops, i didn't remember i deleted this method.